doc/engineering-reference/src/surface-heat-balance-manager-processes/conduction-finite-difference-solution.tex

```diff
@@ -125,95 +219,164 @@ \subsection{Basic Finite Difference Solution Approach}\label{basic-finite-differ
 \end{figure}
 
-The iteration scheme assures that the correct enthalpy, and therefore the correct Cp is used in each time step, and the enthalpy of the material is accounted for accurately. Of course, if the material is regular, the user input constant Cp is used.
+The iteration scheme assures that the correct enthalpy, and therefore the
+correct $C_p$ is used in each time step, and the enthalpy of the material is
+accounted for accurately. Of course, if the material is regular, the user input
+constant $C_p$ is used.
 
-For non-hysteresis finite difference calculations, the algorithm also has a provision for including a temperature coefficient to modify the thermal conductivity. The thermal conductivity is obtained from:
+For non-hysteresis finite difference calculations, the algorithm also has a
+provision for including a temperature coefficient to modify the thermal
+conductivity. The thermal conductivity is obtained from:
 
 \begin{equation}
-k = {k_o} + {k_1}\left( {{T_i} - 20} \right)
+k = k_o + k_1\left( T_i - 20 \right)
 \end{equation}
-
+%
 where:
+\begin{wherelist}
+\item[k_o] is the \SI{20}\celsius\ value of thermal conductivity (normal IDF
+input)
+\item[k_1] is the change in conductivity per degree temperature difference from
+\SI{20}\celsius
+\end{wherelist}
 
-k\(_{o}\) is the 20°C value of thermal conductivity (normal idf~ input)
-
-k\(_{1}\) is the change in conductivity per degree temperature difference from 20°C
-
-As of Version 7, the CondFD implementation was changed to evaluate the thermal conductivity at the interface between nodes, as shown below. In this case, EnergyPlus uses a linear interpolation between nodal points.
+As of Version 7, the CondFD implementation was changed to evaluate the thermal
+conductivity at the interface between nodes, as shown below. In this case,
+EnergyPlus uses a linear interpolation between nodal points.
 
 \begin{equation}
-\medmuskip=0mu
-\thinmuskip=0mu
-\thickmuskip=0mu
-\nulldelimiterspace=0pt
-\scriptspace=0pt
-{C_p}\rho {\rm{\Delta x}}\frac{{T_i^{j + 1} - T_i^j}}{{{\rm{\Delta }}t}} = \frac{1}{2}\left[ {\left( {{k_W}\frac{{\left( {T_{i + 1}^{j + 1} - T_i^{j + 1}} \right)}}{{{\rm{\Delta x}}}} + {k_E}\frac{{\left( {T_{i - 1}^{j + 1} - T_i^{j + 1}} \right)}}{{{\rm{\Delta x}}}}} \right) + \left( {{k_W}\frac{{\left( {T_{i + 1}^j - T_i^j} \right)}}{{{\rm{\Delta x}}}} + {k_E}\frac{{\left( {T_{i - 1}^j - T_i^j} \right)}}{{{\rm{\Delta x}}}}} \right)} \right]
+%\medmuskip=0mu
+%\thinmuskip=0mu
+%\thickmuskip=0mu
+%\nulldelimiterspace=0pt
+%\scriptspace=0pt
+C_p\rho\Delta x\frac{T_i^{j + 1} - T_i^j}{\Delta t} = 
+\frac{1}{2}\left[ \left( k_W\frac{T_{i + 1}^{j + 1} - T_i^{j + 1}}{\Delta x}
+                       + k_E\frac{T_{i - 1}^{j + 1} - T_i^{j + 1}}{\Delta x}
+                 \right)
+                + \left( k_W\frac{T_{i + 1}^j - T_i^j}{\Delta x}
+                       + k_E\frac{T_{i - 1}^j - T_i^j}{\Delta x}\right)\right]
 \end{equation}
-
-Where,
-
+%
+where,
```

Comment:
Any reason you didn't use a wherelist on these items?

Reply:
Yes, this one doesn't give the descriptions in words and the equations could/should be numbered, so I left it as is.

Reply:
Works for me, thanks!